app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,7 @@
  */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+
+.discount_price {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.